drivers/wasapi/audio_driver_wasapi.cpp

```diff
@@ -201,6 +206,26 @@ class CMMNotificationClient : public IMMNotificationClient {
 
 static CMMNotificationClient notif_client;
 
+typedef const char *(CDECL *PWineGetVersionPtr)(void);
+
+bool AudioDriverWASAPI::is_running_on_wine() {
+	HMODULE nt_lib = LoadLibraryW(L"ntdll.dll");
+	if (!nt_lib) {
+		return false;
+	}
+
+	PWineGetVersionPtr wine_get_version = (PWineGetVersionPtr)GetProcAddress(nt_lib, "wine_get_version");
+	if (wine_get_version) {
```

Comment:
Could factor out freeing the lib, and then maybe even skipping the if/else with `return (bool)wine_get_version;`.